app/javascript/tests/patient/monitoring_actions/CaseStatus.test.js

```diff
@@ -7,6 +7,7 @@ import CustomTable from '../../../components/layout/CustomTable';
 import InfoTooltip from '../../../components/util/InfoTooltip';
```

Comment:
Please update this test suite to incorporate tests for the changes you made in this file